tools/build.js

```diff
@@ -2,20 +2,29 @@ var fs = require('fs');
 var path = require('path');
 var stringify = require('json-stable-stringify');
 var commandPath = path.join(__dirname, '..', 'commands.json');
+var redisCommands = require('../');
 
 var Redis = require('ioredis');
 var redis = new Redis(process.env.REDIS_URI);
 
-redis.command(function (err, res) {
+return redis.command().then(function (res) {
   redis.disconnect();
 
-  if (err) {
-    throw err;
-  }
+  // Find all special handled cases
+  var movableKeys = String(redisCommands.getKeyIndexes).match(/case '[a-z-]+':/g).map(function (entry) {
+    return entry.replace(/^case \'|\':$/g, '');
+  });
 
   var commands = res.reduce(function (prev, current) {
+    if (current[2].indexOf('movablekeys') !== -1) {
+      movableKeys.splice(movableKeys.indexOf(current[0]), 1);
```

Comment:
Note to myself: a check, if `current[0]` exists is missing. Otherwise this might result in false positives.